packages/dx-react-grid/docs/reference/table-header-row.md

```diff
@@ -36,10 +36,10 @@ A value with the [TableCellProps](table.md#tablecellprops) shape extended by the
 Field | Type | Description
 ------|------|------------
 column | [Column](grid.md#column) | A column object associated with the header cell.
-allowSorting | boolean | Specifies whether a user can change the associated column's sorting state.
+showSortingControls | boolean | Specifies whether to render controls that toggle the column's sorting state.
```

Comment:
...toggle the column's sorting state.
Can we say '...toggle the column's sorting'? Without 'state'.

Reply:
I'd leave as is, with 'state'.